extensions/wikia/AssetsManager/builders/AssetsManagerSassesBuilder.class.php

```diff
@@ -21,6 +21,10 @@ public function __construct(WebRequest $request) {
 			$builder = new AssetsManagerSassBuilder($sassRequest);
 
 			$this->mContent .= "\n\n/* $file */\n" . $builder->getContent();
+
+			if (strpos($this->mOid, 'ads.scss') !== false) {
```

Comment:
:see_no_evil: we can still do better and try follow @macbre's [commnet](https://github.com/Wikia/app/pull/10652/files#r66602002)